src/infrastructure/validation/testing/TestHarness.java

```diff
@@ -0,0 +1,269 @@
+        Object test = testClass.getDeclaredConstructor().newInstance();
+        boolean result = test.run();  
+         
+        if(result == false){
+          failedNumberOfTests++;
+          if(failedNumberOfTests == 1){
+            logger.log(ModuleID.TEST, LogLevel.INFO, "Failed Tests...");
+          }
+          //log the result of failed test cases
+          logger.log(ModuleID.TEST, LogLevel.INFO, Integer.toString(failedNumberOfTests)+". "+testName+": "+test.getError());
+        }
+        else{
+          successfulNumberOfTests++;
+        }
+      }
+		}
+    catch (Exception e){
+      logger.log(ModuleID.TEST, LogLevel.WARNING, "Exception >> "+e);      	
+    }
+    
+    //result logging 
+    logger.log(ModuleID.TEST, LogLevel.INFO, "\nOverall Result:");
+    logger.log(ModuleID.TEST, LogLevel.INFO, "Total Number of Tests: "+Integer.toString(totalNumberOfTests));
+    logger.log(ModuleID.TEST, LogLevel.INFO, "Total Number of Successful Tests: "+Integer.toString(successfulNumberOfTests));
+    logger.log(ModuleID.TEST, LogLevel.INFO, "Total Number of Failed Tests: "+Integer.toString(failedNumberOfTests));
+  } 
+	 
+	  
+ /**
+  * Boolean method to give result of the Test Case Class
+  * @param testNamePath The absolute path to file name of test case class including .java extension as a string
+  */
+  public void runByName(String testNamePath){
+    int totalNumberOfTests = 0;
+    int successfulNumberOfTests = 0;
+    int failedNumberOfTests = 0;
+    
+    ILogger logger = LoggerFactory.getLoggerInstance();
+    try{
+      totalNumberOfTests = 1;
+
+      //Get full qualified class name from absolute path of testcase
+      String[] arrOfStr = testNamePath.split("src/", 2); 
+      arrOfStr = arrOfStr[1].split(".java", 2);
+      String relPath = arrOfStr[0];
+      String fullQualifiedClassName = relPath.replace("/",".");
+
+      Class<?> testClass = Class.forName(fullQualifiedClassName);
+      Object test = testClass.getDeclaredConstructor().newInstance();
+      boolean result = test.run();
+
+      if(result == false){
+        failedNumberOfTests++;
+        if(failedNumberOfTests == 1){
+          logger.log(ModuleID.TEST, LogLevel.INFO, "\nFailed Test...");
+        }
+        //log the result of failed test cases
+        logger.log(ModuleID.TEST, LogLevel.INFO, Integer.toString(failedNumberOfTests)+". "+testName+" :"+test.getError());
+      }
+      else{
+        successfulNumberOfTests++;
+      }
+    }
+    catch (Exception e){
+      logger.log(ModuleID.TEST, LogLevel.WARNING, "Exception >> "+e);      	
+    }
+      
+    //result logging 
+    logger.log(ModuleID.TEST, LogLevel.INFO, "\nOverall Result:");
+    logger.log(ModuleID.TEST, LogLevel.INFO, "Total Number of Tests: "+Integer.toString(totalNumberOfTests));
+    logger.log(ModuleID.TEST, LogLevel.INFO, "Total Number of Successful Tests: "+Integer.toString(successfulNumberOfTests));
```

Comment:
```suggestion
    logger.log(
       ModuleID.TEST, 
       LogLevel.INFO, 
       "Total Number of Successful Tests: "
       + Integer.toString(successfulNumberOfTests)
    );
```

All the logs can be changed to this format.